### PR TITLE
Fix typos.

### PR DESCRIPTION
**Pull request by waywardmonkeys**

### CODE REVIEW

File: notify/src/windows.rs [hunk 1]
```diff
@@ -138,8 +138,8 @@ impl ReadDirectoryChangesServer {
 
             unsafe {
                 // wait with alertable flag so that the completion routine fires
-                let waitres = WaitForSingleObjectEx(self.wakeup_sem, 100, 1);
-                if waitres == WAIT_OBJECT_0 {
+                let waiters = WaitForSingleObjectEx(self.wakeup_sem, 100, 1);
```
Comment (0xpr03) [inline notify/src/windows.rs:141]: that's intentional

**Review by 0xpr03 — CHANGES_REQUESTED** [review]
Looks good apart from the two lines

File: notify/src/windows.rs (hunk 1 above)
Comment (waywardmonkeys) [reply notify/src/windows.rs]: Whoops, I'd meant to remove those. Sorry about that! Removed these now.

**Comment by 0xpr03** [comment]
Thanks!

**Pull request merged**